scripts/spyder_win_post_install.py

```diff
@@ -87,6 +173,44 @@ def remove():
             winreg.DeleteKey(root, key)
         except WindowsError:
             pass
+        else:
+            print("Successfully removed Spyder shortcuts from Windows "\
+                  "Explorer context menu.", file=sys.stdout)
+    # clean up desktop
+    desktop_folder = get_special_folder_path("CSIDL_DESKTOPDIRECTORY")

```

Comment:
I think this code, i.e. from here to the end of `remove` should be ran only when `is_bdist_wininst == False`, right?